react-autosuggest/react-autosuggest-tests.tsx

```diff
@@ -1,58 +1,85 @@
 // region Imports
 import React = require('react');
 import ReactDOM = require('react-dom');
-import { Autosuggest, SuggestionSelectedEventData } from 'react-autosuggest';
+import * as Autosuggest from 'react-autosuggest';
```

Comment:
Change this to `import Autosuggest = require('react-autosuggest');` since it's a CommonJS exported module